### PR TITLE
Update match-numbers-and-letters-of-the-alphabet.portuguese.md

### PR DESCRIPTION
**Pull request by mffdeo**
I changed the problem comment to Portuguese and added a solution to that problem.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/regular-expressions/match-numbers-and-letters-of-the-alphabet.portuguese.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ let result = myRegex; // Change this line
 <section id='solution'>
 
 ```js
-// solution required
+let quoteSample = "Blueberry 3.141592653s are delicious.";
+let myRegex = /[h-s2-6]/; // 
+let result =quoteSample.match(myRegex); //
```
Comment (nhcarrigan) [inline curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/regular-expressions/match-numbers-and-letters-of-the-alphabet.portuguese.md:55]:
```suggestion
let quoteSample = "Blueberry 3.141592653s are delicious.";
let myRegex = /[h-s2-6]/; // Altere essa linha
let result =quoteSample.match(myRegex); // Altere essa linha
```
Can we add the comments to the solution to match the English version?

**Review by naomi-lgbt — DISMISSED** [review]

File: curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/regular-expressions/match-numbers-and-letters-of-the-alphabet.portuguese.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/regular-expressions/match-numbers-and-letters-of-the-alphabet.portuguese.md:55]: @nhcarrigan Do not add any comments to the non-English curriculum files.   Any code comment that exists in the English file will be automatically translated when other languages are rolled out.  See PR #38040 for more information.

Comment (nhcarrigan) [reply curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/regular-expressions/match-numbers-and-letters-of-the-alphabet.portuguese.md:55]: The provided solution does not appear to pass the challenge tests - one test in particular requires the `g` and `i` flags.

**Comment by RandellDawson** [comment]
Thank you for opening this pull request.

This is a standard message notifying you that we’ve reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you.

Thank you and happy coding.

**Pull request closed without merging**